### ISSUE TITLE
Issue: Pasting in the "Expected Hash Value" field requires selecting the ellipsis

### ISSUE THREAD

**Issue by Sheridan-Tech** [tedsmith/quickhash#96]
Pasting in the "Expected Hash Value" field requires selecting the ellipsis, so rather than having them there why not leave the text field blank?
If it is a validation flag to compare the field, it could be left null or empty and serve the same purpose. This requires the user to select or preemptively delete the "..." when pasting.
I am willing to contribute if needed.

**Comment by tedsmith** [comment]
Fair point. I cant remember off hand why I did it that way, but I'm sure there was\is a reason. I'll keep it in mind for v3.3.2. I'm currently in chaos at home so development is paused until that is sorted.

**Comment by Sheridan-Tech** [comment]
Much appreciated. Let me look now if you want help.
